pandas/core/arrays/base.py

```diff
@@ -610,3 +615,118 @@ def _ndarray_values(self):
         used for interacting with our indexers.
         """
         return np.array(self)
+
+
+
+class ExtensionOpsMixin(object):
+    """
+    A base class for linking the operators to their dunder names
+    """
+    @classmethod
+    def addArithmeticOps(cls):
```

Comment:
please use PEP8 and make it private (`_add_arithmetic_ops`)

Reply:
Done.  Somehow flake8 didn't pick it up for me when I ran it this time.